scipy/optimize/_differentialevolution.py

```diff
@@ -447,26 +576,39 @@ def solve(self):
 
         nfev, nit, warning_flag = 0, 0, False
         status_message = _status_message['success']
+        
+        # calculate energies to start with for the whole population
+        parameters = []
+        for candidate in self.population:
+            # incapsulate additional fixed arguments to parameters
+            parameters.append(self._scale_parameters(candidate))
 
-        # calculate energies to start with
-        for index, candidate in enumerate(self.population):
-            parameters = self._scale_parameters(candidate)
-            self.population_energies[index] = self.func(parameters,
-                                                        *self.args)
-            nfev += 1
 
-            if nfev > self.maxfun:
-                warning_flag = True
-                status_message = _status_message['maxfev']
-                break
+        # evaluate function for the whole population using a pool
+        if len(self.args) is not 0:
```

Comment:
Lines 588 through 591 can be replaced by:

    energies = self.pool_map(unpacker(self.func), parameters, itertools.repeat(args, np.size(parameters, 0)))
    self.population_energies = np.array(energies)

You'll need to add to add the following function to the code, either inline or as a separate static function:

    def unpacker(func):
        def inner(x, args):
            if not len(args):
                return func(x)
            else:
                return func(x, *args)
        return inner